install/kubernetes/cilium/templates/hubble-relay-deployment.yaml

```diff
@@ -1,81 +1,87 @@
+{{- if .Values.hubble.relay.enabled }}
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ .Chart.Name }}
-  namespace: {{ .Release.Namespace }}
+  name: hubble-relay
   labels:
-    k8s-app: {{ .Chart.Name }}
+    k8s-app: hubble-relay
+  namespace: {{ .Release.Namespace }}
 spec:
-  replicas: {{ .Values.numReplicas }}
+  affinity:
```

Comment:
this `affinity` seems not belonged to Deployment in the API Deployment spec

Reply:
correct it is in the pod template spec, thanks